lib/formatter.js

```diff
@@ -28,10 +28,18 @@ module.exports = function formatMessages(results) {
         messageType = chalk.yellow(messageType);
       }
 
+      let line, column;
+      if (message.stylelintType === 'invalidOption') {
```

Comment:
I've been trying to avoid baking any specific knowledge of stylelint here if possible. Could we just default to `''` if `message.{line,column}` are non-numeric?

Reply:
yep